packages/messenger/MessageHandler/RetryFailedMessageMessageHandler.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Draw\Component\Messenger\MessageHandler;
+
+use Draw\Component\Messenger\Message\RetryFailedMessageMessage;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Messenger\Attribute\AsMessageHandler;
+
+class RetryFailedMessageMessageHandler
+{
+    public function __construct(
+        private KernelInterface $kernel,
+    ) {
+    }
+
+    #[AsMessageHandler]
+    public function handleRetryFailedMessageMessage(RetryFailedMessageMessage $message): void
+    {
+        $application = new Application($this->kernel);
```

Comment:
Run this in command line (using process) to make sure it's the same execution context that we would have and that the execution "scope" is the same. We could have a state that is affecting execution right now